.circleci/config.yml

```diff
@@ -0,0 +1,81 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:18-git
+    environment:
+      APP_NAME: "get-access/cla_backend"
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Login to container registry
+          command: |
+            apk add --no-cache --no-progress --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing/ aws-cli
+            login="$(aws ecr get-login --region eu-west-1 --no-include-email)"
+            ${login}
+      - run:
+          name: Build Docker image
+          command: |
+            docker build --tag $ECR_ENDPOINT/$APP_NAME:$CIRCLE_SHA1 .
+            docker tag $ECR_ENDPOINT/$APP_NAME:$CIRCLE_SHA1 $ECR_ENDPOINT/$APP_NAME:$CIRCLE_BRANCH
+      - run:
+          name: Validate Python version
+          command: |
+            docker run --rm --tty --interactive $ECR_ENDPOINT/$APP_NAME:$CIRCLE_SHA1 python --version | grep "2.7"
+      - run:
+          name: Push Docker image
+          command: |
+            docker push $ECR_ENDPOINT/$APP_NAME:$CIRCLE_SHA1
+            docker push $ECR_ENDPOINT/$APP_NAME:$CIRCLE_BRANCH
+
+  test:
+    docker:
+      - image: circleci/python:2.7
+        environment:
+          DB_NAME: circle_test
+          DATABASE_URL: postgresql://root@localhost
+          DJANGO_SETTINGS_MODULE: cla_backend.settings.circle
+      - image: circleci/postgres:9.4.18-alpine
```

Comment:
_nitpick_ It's probably not necessary to mention the patch version, e.g. we could use `9.4-alpine` as the tag, so we can avoid updating this file too many times.